src/components/form/web-forms-settings-confirmation.vue

```diff
@@ -102,6 +106,12 @@ const setWebformsEnabled = () => {
 };
 </script>
 
+<style lang="scss">
+  #modalBanner {
```

Comment:
I could imagine another component using an `id` named `modalBanner`. CSS isn't scoped to components: once loaded here, it will apply anywhere in the app. How about either: (1) renaming this id (e.g., to `form-web-forms-settings-confirmation-banner`), or (2) moving this style to the `Modal` component (e.g., `.modal-banner img { width: 100%; }`).